src/Tizen.NUI/src/public/Window/BorderWindow.cs

```diff
@@ -168,6 +169,29 @@ internal void UpdateProperty()
                         AddAuxiliaryHint("wm.policy.win.resize_aspect_ratio", "1");
                     }
                 }
+                if (borderLineThickness != borderInterface.BorderLineThickness)
+                {
+                    int diffBorderLine = (int)(borderInterface.BorderLineThickness - borderLineThickness);
+                    borderLineThickness = borderInterface.BorderLineThickness;
+
+                    if (borderView != null)
+                    {
+                        Extents extents = borderView.Padding;
+                        ushort start = (extents.Start + diffBorderLine) > 0 ? (ushort)(extents.Start + diffBorderLine) : (ushort)0;
+                        ushort end = (extents.End + diffBorderLine) > 0 ? (ushort)(extents.End + diffBorderLine) : (ushort)0;
+                        ushort top = (extents.Top + diffBorderLine) > 0 ? (ushort)(extents.Top + diffBorderLine) : (ushort)0;
+                        ushort bottom = (extents.Bottom + diffBorderLine) > 0 ? (ushort)(extents.Bottom + diffBorderLine) : (ushort)0;
+                        borderView.Padding = new Extents(start, end, top, bottom);
+                        if (IsMaximized() == true)
+                        {
+                            borderView.OnMaximize(true);
+                        }
+                    }
+
+                    ResizedEventArgs e = new ResizedEventArgs();
+                    e.WindowSize = WindowSize;
+                    OnBorderWindowResized(this, e);
```

Comment:
Is this code intended to call internal function only?
seems below code is more general for resize event..
```suggestion
                    Resized?.Invoke(this, e);
```